src/tree.ts

```diff
@@ -513,9 +513,12 @@ export class ProjectOutlineProvider implements vscode.TreeDataProvider<BaseNode>
     existing.updateCodeModel(model, {...ctx, nodesToUpdate: updates, folder});
 
     this._changeEvent.fire(null);
-    for (const node of updates) {
-      this._changeEvent.fire(node);
-    }
+    // i have no idea why this was added.
```

Comment:
Can you just delete this instead of commenting it out?  I looked at the interface and it says that firing the event with `null` causes a full tree update which is all we need to do.

Reply:
deleted (https://github.com/microsoft/vscode-cmake-tools/pull/1169/commits/8e165cf4815358a421359be12633e3e4beedf58c , https://github.com/microsoft/vscode-cmake-tools/pull/1169/commits/3eaeac6f9e464034afcc199d95af712b0e81f849 )